recipes/glsl-optimizer/all/conanfile.py

```diff
@@ -0,0 +1,38 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+
+class GLSLOptimizerConan(ConanFile):
+    name = "glsl-optimizer"
+    description = "GLSL optimizer based on Mesa's GLSL compiler. Used in Unity for mobile shader optimization."
+    homepage = "https://github.com/aras-p/glsl-optimizer"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "glsl", "opengl")
+    license = "MIT"
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake"
+    exports_sources = "CMakeLists.txt"
+
```

Comment:
```suggestion
    options = {
        "fPIC": [True, False],
    }
    default_options = {
        "fPIC": True,
    }
```

It does not seem to export symbols. Can you try a shared configuration with `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS` on Windows ?

Reply:
I'm not exactly sure what do you mean by this, it seems like linking goes correctly even without it?
Added the flag anyways.

Reply:
On Windows, when building a dll, you have to explicitly export symbols to create a import static library+dll.

Reply:
@hnOsmium0001 take a look: https://github.com/conan-io/conan-center-index/blob/ccd4ba40a2f39d03108c3bb25bae80c1410e8114/recipes/kcp/all/CMakeLists.txt#L7